### PR TITLE
142 solution reply post

### PR DESCRIPTION
**Pull request by Tguntenaar**
Solves issue #142

[Preview version](https://near.org/thomasguntenaar.near/widget/gigs-board.pages.Feed)

-User sees the same fields & preview in the solution reply as shown in the create solution post
-User is able to select USDC and proceed successfully without a warning message

### CODE REVIEW

**Comment by ailisp** [comment]
Hmm when I try reply from your preview, it's still not showing USDC:


image

Did you deploy something else that overrides this branch?

**Comment by Tguntenaar** [comment]
@ailispI noticed that the previous preview was accidentally overwritten, I apologize for the late reaction. Here's the updated preview for your review [preview](https://near.org/thomasguntenaar.near/widget/gigs-board.pages.Feed)

File: src/gigs-board/entity/post/PostEditor.jsx [hunk 1]
```diff
@@ -110,15 +113,9 @@ if (!state.draftStateApplied && props.draftState) {
...
+  Submission: ["name", "description", "fund_raising"],
   Attestation: ["name", "description"],
-  Sponsorship: [
-    "name",
-    "description",
-    "amount",
-    "sponsorship_token",
-    "supervisor",
-  ],
+  Sponsorship: ["name", "description", "fund_raising"],
```
Comment (ailisp) [inline src/gigs-board/entity/post/PostEditor.jsx:118]: Is this intentional? I saw you proposed to remove this in the issue, but @ori-near hasn't reply so is this decided.

File: src/gigs-board/entity/post/PostEditor.jsx [hunk 2]
```diff
@@ -142,7 +139,12 @@ const onSubmit = () => {
     },
     Submission: {
       name: state.name,
-      description: state.description,
+      description: generateDescription(
+        state.description,
+        state.amount,
+        state.token,
+        state.supervisor
+      ),
```
Comment (ailisp) [inline src/gigs-board/entity/post/PostEditor.jsx:147]: If really to remove fields from sponsorship, below Sponsorship:
```
 Sponsorship: {
      name: state.name,
      description: state.description,
      amount: state.amount,
      sponsorship_token: state.token,
      supervisor: state.supervisor,
      sponsorship_version: "V1",
    },
```

 should also update, and call contract to create sponsorship should also update. I suggest to consider & address sponsorship in a future PR. in this PR, only do fix #142.

**Review by ailisp — COMMENTED** [review]
Preview looks good! but sponsorship I don't recommend to change in this PR

File: src/gigs-board/entity/post/PostEditor.jsx (hunk 1 above)
Comment (Tguntenaar) [reply src/gigs-board/entity/post/PostEditor.jsx:118]: Yes I spoke with @ori-near on a call about this.

**Pull request merged**